spanner/tests/system/utils/scrub_instances.py

```diff
@@ -0,0 +1,24 @@
+from google.cloud.spanner import Client
+from .streaming_utils import INSTANCE_NAME as STREAMING_INSTANCE
+
+STANDARD_INSTANCE = 'google-cloud-python-systest'
+
+
+def scrub_instances(client):
+
```

Comment:
Empty line just for fun? Maybe just instantiate the `Client` inside this function?

Reply:
8582a76

Instantiating the client inside `if __name__ == "__main__"` is what the other utility scripts do.